src/pytest_postgresql/factories.py

```diff
@@ -225,7 +225,6 @@ def postgresql(
     :param db_name: database name
     :param load: SQL to automatically load into our test database
     :param isolation_level: optional postgresql isolation level
-                            defaults to ISOLATION_LEVEL_AUTOCOMMIT
```

Comment:
Keep that info, and what's more, I'd still keep within here an or to enforce it. Otherwise we loose the information on what's the default, since that function is not really related to the janitor itself and we can't say: for kwargs information look at janitor.

Reply:
Restored the line.
Sorry, but what 'or' are you referring to? If it's [this one](https://github.com/ClearcodeHQ/pytest-postgresql/pull/370/files#diff-47a715ffd93e4dc187da7b1a66fb6f41b7dc05427479c92d61409665dec52dc0R46), it won't work here: unless I am mistaken, pytest immediately parses the function body, so a reference to `psycopg2.*` will break it.

Reply:
function bodies are not parsed, unless being executed. If it would be the case then [this](https://github.com/ClearcodeHQ/pytest-postgresql/blob/master/src/pytest_postgresql/factories.py#L241) wouldn't work either.

Reply:
The body of the function is executed by pytest (it locates and integrates plugins on startup, as far as i understand), it's called from here https://github.com/pytest-dev/pytest/blob/master/src/_pytest/assertion/rewrite.py#L129
Referring to `psycopg2` works because that identifier exists (see #369), but it points to a boolean, not a module.